cluster/kube-push.sh

```diff
@@ -27,10 +27,60 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/cluster/kube-env.sh"
 source "${KUBE_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
-echo "Updating cluster using provider: $KUBERNETES_PROVIDER"
+function usage() {
+  echo "${0} [-m|-n <node id>] <version>"
```

Comment:
Can we drop the individual node support for now, or label it as experimental? It's a matter of shipping features I don't necessarily want to support for eternity.